svm/src/account_loader.rs

```diff
@@ -1408,26 +1400,6 @@ mod tests {
         assert_eq!(shared_data, expected);
     }
 
-    #[test]
```

Comment:
`account_shared_data_from_program()` now always returns a value, not a `Result`.